core/trino-main/src/main/java/io/trino/sql/planner/SubqueryPlanner.java

```diff
@@ -268,7 +268,7 @@ private PlanBuilder planScalarSubquery(PlanBuilder subPlan, Cluster<SubqueryExpr
                 subPlan,
```

Comment:
from commit message:

> Additionally, TransformCorrelatedJoinToJoin was completing
with TransformCorrelatedScalarSubquery to decorrelate
EnforceSingleRowNode.


did you mean  completing or competing ?